### ISSUE TITLE
bug(mat-progress-bar): ChromeVox can't read a progress bar

### ISSUE THREAD

**Issue by jamOne-** [angular/components#22165]
#### Reproduction

Steps to reproduce:
1. Go to link https://material.angular.io/components/progress-bar/overview#determinate.
2. Enable ChromeVox.
3. Try to focus the progress bar with the screen reader.
 
#### Expected Behavior

ChromeVox should announce progress bar element with its state.

#### Actual Behavior

ChromeVox skips this element while navigating through a page.

#### Environment

- Browser(s): Chrome 88.0.4324.186 (Official Build) (64-bit)
- Operating System (e.g. Windows, macOS, Ubuntu): Chrome OS

#### Additional note

When I add aria-hidden="true" attribute to every child of `mat-progress-bar` element the problem disappears.
I tried filing a bug against Chrome OS team [link](https://bugs.chromium.org/p/chromium/issues/detail?id=1095603), but haven't received any response since Jun, so decided to try the other way around, with making a change to `angular/components`.
